.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ default_stages: [commit]
 repos:
     # Check formatting and lint for starlark code
     - repo: https://github.com/keith/pre-commit-buildifier
-      rev: 5.0.1
+      rev: 6.1.0.1
```

Comment:
super semver

Reply:
Alex made the same change when adding this buildifier ci step in the rules_template 🤷